docker-accumulo/accumulo-conf/accumulo-site.xml

```diff
@@ -127,7 +127,7 @@
 
   <property>
     <name>tserver.walog.max.size</name>
-    <value>256M</value>
+    <value>100M</value>
```

Comment:
can you document these changes in a <git root>/TUNING?

also - these seem like very minor changes to memory usage that I'm surprised they make a difference. did this crash because the accumulo jvm heap run out of memory? if the latter, maybe we should just increase the java heap size for the accumulo JVM instead?